integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonScanRDD.scala

```diff
@@ -90,6 +90,8 @@ class CarbonScanRDD[T: ClassTag](
   }
   private var vectorReader = false
 
+  private var directScan = false
```

Comment:
sometimes in the code, it is called `directFill` but sometimes it is called `directScan`

Reply:
ok will change to directFill